scripts/docker/fedora.24/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM fedora:24
 
-RUN dnf install -y bash git cmake wget which python clang-3.8.1-1.fc24.x86_64 llvm-devel-3.8.1-2.fc24.x86_64 make libicu-devel lldb-devel.x86_64 \
+RUN dnf install -y bash git cmake wget which python clang-3.8.0-1.fc24.x86_64 llvm-devel-3.8.0-1.fc24.x86_64 make libicu-devel lldb-devel.x86_64 \
```

Comment:
> Given the above, do you want to fix Fedora 23 dockerfile as well?

I think that should be done as part of developing a comprehensive toolset story for these Docker images, as mentioned in https://github.com/dotnet/core-setup/pull/1383#issuecomment-276462691.  This can be done as part of https://github.com/dotnet/core-eng/issues/225 or https://github.com/dotnet/core-eng/issues/365, which I will be working on this sprint.